examples/query-suggestions-history-with-category/app.tsx

```diff
@@ -0,0 +1,33 @@
+import { autocomplete } from '@algolia/autocomplete-js';
+import { createQuerySuggestionsPlugin } from '@algolia/autocomplete-plugin-query-suggestions';
+import { createLocalStorageRecentSearchesPlugin } from '@algolia/autocomplete-plugin-recent-searches';
+import algoliasearch from 'algoliasearch';
+
+import '@algolia/autocomplete-theme-classic';
+
+const appId = 'latency';
+const apiKey = '6be0576ff61c053d5f9a3225e2a90f76';
+const searchClient = algoliasearch(appId, apiKey);
+
+const recentSearchesPlugin = createLocalStorageRecentSearchesPlugin({
+  key: 'qs-with-rs-example',
+  limit: 5,
+});
+
+const querySuggestionsPlugin = createQuerySuggestionsPlugin({
+  searchClient,
+  indexName: 'instant_search_demo_query_suggestions',
+  getSearchParams() {
+    return {
+      hitsPerPage: 5,
+    };
+  },
+  categoryAttribute: ['categories'],
```

Comment:
With the latest `next` branch, you need to provide the full path:

```suggestion
  categoryAttribute: [
    'instant_search',
    'facets',
    'exact_matches',
    'categories',
  ],
```

(Perhaps it's working locally because the example has cached the Autocomplete library. In that case, you can remove the `dist` and `.cache` folder and `start` the example again.)